### PR TITLE
CSS to SCSS and Gulp

### PR DESCRIPTION
**Pull request by kingsleyzissou**
So navigate into the folder in your terminal and enter these commands:

Step 1: npm install
Step 2: gulp watch
Step 3: go edit one of the scss files and change the style, gulp will recompile and the browser will update automatically with the changes made.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,79 @@
-.DS_Store
+.### OSX ###
```
Comment (dimitraz) [inline .gitignore]: Some of this looks a bit unnecessary lol

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        },
+    })
+});
+
+gulp.task('clean', function () {
+    return del.sync('dist');
+});
+
+gulp.task('build', [`clean`, `sass`], function () {
+    console.log('Building files');
```
Comment (dimitraz) [inline gulpfile.js]: Would it be possible to also add a minifying task as well? so like build the css and min.css?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (dimitraz) [inline index.html]: Does this index.html need to be checked in? we can prob leave it in the gitignore as its more of a dev dep

File: .gitignore (hunk 1 above)
Comment (kingsleyzissou) [reply .gitignore]: Oh lol, yeah that was autogenerated by .vscode

File: gulpfile.js (hunk 2 above)
Comment (kingsleyzissou) [reply gulpfile.js]: Yes, it is possible. I'll get onto that

File: index.html (hunk 3 above)
Comment (kingsleyzissou) [reply index.html]: True. You'll just have to make sure it's in the folder yourself

File: gulpfile.js (hunk 2 above)
Comment (dimitraz) [reply gulpfile.js]: 💯

File: .gitignore [hunk 4]
```diff
@@ -29,51 +29,12 @@ logs
 *.log
 npm-debug.log*
```
Comment (dimitraz) [inline .gitignore]: Can you delete lines https://github.com/kingsleyzissou/breakpoint.css/blob/2d8663ccff82530aef09496006257c3502b93d9c/.gitignore#L3-L30 
and .npm

File: gulpfile.js [hunk 5]
```diff
@@ -8,6 +8,7 @@ gulp.task('sass', function () {
     return gulp.src('src/breakpoint.scss')
         .pipe(notify("Compiling SASS!"))
         .pipe(sass()) 
+        .pipe(sass({ outputStyle: 'compressed' }))
```
Comment (dimitraz) [inline gulpfile.js]: can we rename the output to breakpoint.min.css?

File: index.html [hunk 6]
```diff
@@ -8,6 +8,16 @@
...
+            <div class="item">1</div>
+            <div class="item">2</div>
+            <div class="item">3</div>
+            <div class="item">4</div>
+            <div class="item">5</div>
+            <div class="item">6</div>
+        </div>
+    </div>
 </body>
 </html>
```
Comment (dimitraz) [inline index.html]: I think you need to delete this file, its in the gitignore but still checked in to git

**Pull request merged**